docs/pages/development/installation.md

```diff
@@ -68,15 +68,19 @@ No additional changes are needed to install the package on Android. 🎉
 
 ## 2. Basic configuration
 
-Development builds use deep links to open projects from the QR code. If you have added a custom deep link scheme to your project, your development build will use it. However, if this isn't the case, you need to configure the deep link support for your app. The `uri-scheme` package will do this for you once you have chosen a scheme.
+Development builds use deep links to open projects via a QR code. If you have added a custom deep link scheme to your project, your development build will use it. However, if this isn't the case, you need to configure deep link support for your app. The `uri-scheme` package will do this for you, once you have chosen a scheme. 
```

Comment:
```suggestion
Development builds use deep links to open projects via a QR code. If you are adding a custom deep link scheme to your project, your development build will use it. However, if this isn't the case, you need to configure deep link support for your app. Once you have chosen a scheme, the `uri-scheme` package will do this for you. 
```

Reply:
I don't understand the "have added" vs "are adding" suggestion, can you elaborate @amandeepmittal

Reply:
Was just trying to avoid any passive voice usage. This is something I am trying to push for all new docs or updates to the existing docs. 

Now you point it out, I think you meant that the custom deep link scheme already exists in the project. Feel free to revert the suggestion.

I also think since this documentation is following a "path" for the user to create a new project and then add the expo-dev-client package, why would they have a custom deep linking set up before? Are there any scenarios/use cases where it may be implied that the custom deep link scheme is already configured?

Reply:
I imagine that deep linking can be configured in a standalone RN project, beyond the scope of expo-dev-client (you want a link from an article or your website to deep link a user into a nested view within your app). This guide is meant for readers who are both creating new projects or have an existing project. Does that address your question?

Reply:
Yes, it does. Thank you for explaining the context of "existing apps".